packages/components/src/custom-gradient-picker/index.js

```diff
@@ -53,7 +53,7 @@ const GradientAnglePicker = ( { gradientAST, hasGradient, onChange } ) => {
 		<AnglePickerControl
 			__nextHasNoMarginBottom
 			onChange={ onAngleChange }
-			labelPosition="top"
```

Comment:
This prop didn't seem to have any effects, judging by `AngleControlPicker`'s code